tests/integrations/huggingface_hub/test_huggingface_hub.py

```diff
@@ -15,8 +15,10 @@
 def mock_client_post(client, post_mock):
     # huggingface-hub==0.28.0 deprecates the `post` method
     # so patch `_inner_post` instead
-    client.post = post_mock
-    client._inner_post = post_mock
+    if hasattr(client, "post"):
```

Comment:
This is just a small improvement that i did on the fly. Because in `huggingface-hub>=0.28.0` there is no `.post` attribute and it would be created in the tests.